### PR TITLE
Fixes #16 and #17

### LINKED ISSUE #16: Add "How to override css" in README

### PR DESCRIPTION
**Pull request by lm-n**
- Includes example where accessible output is visible. #17 
- Includes instructions on how to override css. #16
- Updates colornamer.js cdn on gruntfile.js
- Updates contributors

### CODE REVIEW

**Comment by lm-n** [comment]
@MathuraMG can you take a look at this?

**Comment by MathuraMG** [comment]
@lm-n  can we change the example name to something smaller ? Like exampleStyled?

File: README.md [hunk 1]
```diff
@@ -18,15 +18,27 @@ It helps create 3 kinds of output
...
 
-To get the **table text output**, include the following in your HTML file, where you want the output to be available.
+To get the **table text output**, include the following in your HTML file, and place it where you want the output to be available:
 `<section id="gridOutput-content"></section>`
 
-To get the **sound output**, include the following in your HTML file, where you want the output to be available.
+To get the **sound output**, include the following in your HTML file, and place it where you want the output to be available:
 `<section id="soundOutput-content"></section>`
 
+To make the **plain text and table text outputs** visible include the following in your HTML file:
```
Comment (MathuraMG) [inline README.md]: The formatting for this seems to be off in readme. Should it be in a code bloock ``` ?

Comment (lm-n) [reply README.md]: I've made this change

**Comment by lm-n** [comment]
@MathuraMG I've Changed the name and the READ.ME

**Comment by MathuraMG** [comment]
Sweet! Let's merge it :D

**Pull request merged**